### DISCUSSION TITLE
Security Onion IDH Custom Webpage

### DISCUSSION THREAD

**Discussion by Tredmond7511** [Security-Onion-Solutions/securityonion#9005] [Unsupported Versions] [answered]
We have stood up the http server on our security onion IDH node. It works great only issue we have is that the web page is not very convincing for our environment. Furthermore our red team is experienced in security onion and will likely recognize it immediately. I saw on the docs that there is a way to upload custom html skins but nothing in the way of actual procedures.

**Comment by dougburks** [comment, marked as answer]
This will require some more work on our side:
https://github.com/Security-Onion-Solutions/securityonion/issues/9045
